src/buildercore/terraform.py

```diff
@@ -9,7 +9,7 @@
 from functools import reduce
 
 EMPTY_TEMPLATE = '{}'
-PROVIDER_FASTLY_VERSION = '0.1.4',
+PROVIDER_FASTLY_VERSION = '0.4.0',
```

Comment:
Add in `0.3.0`, but we should use the more recent version to use snippets too.

Reply:
There are always [bugfixes](https://github.com/terraform-providers/terraform-provider-fastly/blob/master/CHANGELOG.md#040-october-02-2018) so I usually go for the latest stable, if `0.x` can be considered stable